src/components/Mailbox.vue

```diff
@@ -329,13 +328,7 @@ export default {
 				return
 			}
 
-			let next
-			if (idx === 0) {
-				next = envelopes[idx + 1]
-			} else {
-				next = envelopes[idx - 1]
-			}
-
+			const next = this.envelopes[idx === 0 ? idx : idx - 1]
```

Comment:
should it be `idx === 0 ? idx + 1 : idx - 1` here?

Reply:
or just `1` instead of `idx + 1`

Reply:
No, that was one of the bugs I found 😊 Since we removed the 0th item of the array, the new 0th item would be the desired message, while the 1st would be one down the list. Not sure if that sentence made sense to anyone other than me 😅